streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBStoreTest.java

```diff
@@ -76,6 +76,25 @@ public void tearDown() throws Exception {
     }
 
     @Test
+    public void shouldNotThrowExceptionOnRestoreWithBulkLoad() throws Exception {
```

Comment:
Looked into it further and the test does work. Not sure why it didn't work, but it is reliably working now